cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/permission/DefaultPermissionManagement.java

```diff
@@ -85,7 +86,7 @@ public boolean testPermissible(@Nullable IPermissible permissible) {
     @Override
     @NotNull
     public Collection<IPermissionGroup> getGroups(@Nullable IPermissible permissible) {
-        Collection<IPermissionGroup> permissionGroups = new ArrayList<>();
+        List<IPermissionGroup> permissionGroups = new ArrayList<>();
```

Comment:
Reduce of creation of new arrays in the memory. 
Try with clear and a class based array. 
Java saves for you the reference.